src/background/responders.ts

```diff
@@ -50,12 +50,51 @@ function updateActiveFeedback(state: StoreState, callback: (feedbackTarget: Feed
   return updateFeedback(state, target, callback(target))
 }
 
+const standardAlertEnding =
```

Comment:
I think saving `alertHTML` instead of alert strings is a risk we shouldn't take. At the moment, the `alertHTML` is completely handled by the client, but if one day we decide to fetch it from the server, it becomes a security risk. We can add the "support" message for every alert or save a "shouldShowSupportMessage" flag in the state.

Reply:
great idea, done.